addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -144,6 +144,15 @@ html[lang] > body.editor_enable [data-oe-translation-state] {
     }
 }
 
+.o_scroll_button_opt {
+    & > we-title {
+        margin-right: auto !important;
+    }
+    & > div {
+        flex-grow: 0 !important;
+    }
+}
```

Comment:
Mmh in fact just a `justify-content: flex-end;` missing on `we-row > div` I think, I'll fix this, you can remove this from your branch